deidentification/index.md

```diff
@@ -0,0 +1,8 @@
+## De-identification
+
+The process of de-identification is removing identifiers from a dataset, such cities, dates, names, ID's.
```

Comment:
```suggestion
De-identification is the process of making a dataset less identifiable or even anonymous. This can consist of completely removing identifiable information, generalizing data points (e.g., replace birth date with age), replacing identifiable values with non-sensitive ones, etc.
```